requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django~=3.2.10
```

Comment:
Make sure to mention the devs env setup in the readme.

Reply:
@Martolivna did you reopen all the threads manually? Usually, you don't need to do this in case when GitHub closed them automatically. I was mostly talking about the cases when humans close threads, but nothing appears to be addressed in them.

Although, it's still useful to add commit links when it's unobvious how something changed. Anyway… Marking this as resolved too.

Reply:
@webknjaz I reopened it because I had previously closed it myself, and I wasn't aware that it could be closed automatically after that.

Reply:
By the way, can you see new messages in resolved conversations?

Reply:
@Martolivna I only see them on the UI if I find the right one and expand it. But I also get email notifications with direct links to those messages.
The automatic close only happens if you commit a change to the line with the conversation attached, I think. Or if you accept a suggested code change via GitHub UI.